usr/lib/ep11_stdll/new_host.c

```diff
@@ -106,13 +106,26 @@ CK_RV ST_Initialize(API_Slot_t *sltp, CK_SLOT_ID SlotNumber,
     bt_init(&sltp->TokData->publ_token_obj_btree, call_object_free);
 
     if (strlen(sinfp->tokname)) {
-        sprintf(abs_tokdir_name, "%s/%s", CONFIG_PATH, sinfp->tokname);
-        TRACE_DEVEL("Token directory: %s\n", abs_tokdir_name);
-        init_data_store(sltp->TokData, (char *) abs_tokdir_name,
-                        sltp->TokData->data_store);
+        int snres = snprintf(abs_tokdir_name, PATH_MAX, "%s/%s",
```

Comment:
same comment as for the other ST_Initialize functions in the other new_host.c files. 
Also this one is slightly different to the previous. 
If we duplicate code, then we should at least make sure that the duplicated code is the same as much as possible. This makes it easier to keep the copies in sync.

Reply:
Moved the declaration to the top of the function and kept assignment here.